analysis2_manuscript.tex

```diff
@@ -420,15 +423,14 @@ \section*{Example Applications with Selection}
     For this simulation we have used
     a genetic map from the HapMap Project \citep{international2007second} along with a
     DFE for deleterious alleles (estimated by \cite{huber2017determining})
-    that acts on exonic positions defined from the HAVANA group release 104 for the human genome.
+    that acts on exonic \kelcomment{do you mean exonic, or nonsynonymous?} positions defined from the HAVANA group release 104 for the human genome.
```

Comment:
so selected mutations are falling on exons- there is no sense of synonymous or nonsyn sites within the simulations